DigitalPassaport/DigitalPassaport/Models/Pass.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Pass.swift
+//  DigitalPassaport
+//
+//  Created by Milena Predic on 18.8.23..
+//
+
+import UIKit
+
+struct Pass: Codable {
```

Comment:
If I'm correct you use this codable inside viewModels and views, which is wrong. What then the purpose of repository? Didn't I mention that we always should map data, and won't use decodable inside views?